modules/compute-vm/variables.tf

```diff
@@ -35,14 +35,21 @@ variable "attached_disk_defaults" {
   }
 }
 
+variable "boot_disk_snapshot_policy" {
```

Comment:
can we move this to the `boot_disk` variable?

Reply:
much better!